README.md

```diff
@@ -35,39 +36,42 @@ Then you can setup your workspace to build this application by running one of
 the following:
 
 For vanilla Zephyr:
-```
+
+```bash
 west init -m https://github.com/golioth/golioth-memfault-zephyr-example.git
 ```
 
 For nRF Connect SDK (NCS):
-```
+
+```bash
 west init -m https://github.com/golioth/golioth-memfault-zephyr-example.git --mf west-ncs.yml
 ```
 
 To build the application (see below for a list of supported boards):
 
-```
-west build -p -b <board_name> app
+```bash
+west build -p -b <board_name> memfault-golioth-example
```

Comment:
The manifest sets
``` 
self:
    path: app
```
so this we should still use the `app` path here. Alternatively, we could change the path for the repo in the manifest, but that could be confusing since we're naming the repo differently in each org.